pkg/apis/flyteworkflow/v1alpha1/node_status.go

```diff
@@ -80,20 +80,16 @@ func (s *DynamicNodeStatus) Equals(o *DynamicNodeStatus) bool {
 	return s.Phase == o.Phase
 }
 
-type SubWorkflowNodeStatus struct {
-	Phase WorkflowPhase `json:"phase"`
-}
-
-func (s SubWorkflowNodeStatus) GetPhase() WorkflowPhase {
-	return s.Phase
-}
+type WorkflowNodePhase int
 
-func (s *SubWorkflowNodeStatus) SetPhase(phase WorkflowPhase) {
-	s.Phase = phase
-}
+const (
+	WorkflowNodePhaseNone WorkflowNodePhase = iota
+	WorkflowNodePhaseExecuting
+)
 
 type WorkflowNodeStatus struct {
-	WorkflowName string `json:"name"`
+	WorkflowName string            `json:"name"`
```

Comment:
do we need to store the workflowname ? or is it just for debugging. If so we can generate it using kubectl flyte

Reply:
removed.